docs/commands.md

```diff
@@ -450,6 +476,89 @@ The returned array will contain key1,labels1,values1,...,keyN,labelsN,valuesN, w
          2) "20"
 ```
 
+##### Query time series with metric=cpu, group them by metric_name reduce max
+
+```sql
+127.0.0.1:6379> TS.ADD ts1 1 90 labels metric cpu metric_name system
```

Comment:
@filipecosta90 lets use real timestamp to avoid confusion and to be more consistent with our millis story.

Reply:
:+1: agree. proceed with the changes.